docs/content_management/create_edit_pages.md

```diff
@@ -157,12 +157,23 @@ The following actions are available:
 - Move up block - allows you to change position of the block on the page by moving it up
 - Move down block - allows you to change position of the block on the page by moving it down
 - Block settings - allows you to access properties window
+- Copy - allows you to reuse a configured block across pages by using the Clipboard section.
 - Duplicate - duplicates existing block with its settings
 - Refresh - refreshes preview of the block
 - Delete - deletes existing block
 
 ![Block window - size adjustment](img/block_window_size_adjust.png "Block window - size adjustment")
 
+#### Copy blocks
+
+Copying blocks allows you to reuse configured blocks across different pages.
+
+When you execute the "Copy" action on a block, it creates a new entry in the "Clipboard" section of the Page Blocks toolbox. Inserting the block from the clipboard adds an exact replica of the block, including its configuration, as it existed at the time of copying.
```

Comment:
```suggestion
When you copy a block, it gets saved in the "Clipboard" section of the Page Blocks toolbox.
Inserting the block from the clipboard puts an exact copy of it, with all its settings, as it was when you copied it.
```
How exactly do you insert a block? I'd focus on the steps, not the behavior of the system, something along the lines of:
"To do it, in the block's toolbar, click Copy. Then, in the Page Blocks toolbox, copy the content of the Clipboard section. You can now switch to another page and insert the block by..."

Reply:
You can drag and drop the block from the clipboard section to the page - exactly like the other blocks (the only difference is that it's inserted preconfigured).